zebra-consensus/src/parameters/subsidy.rs

```diff
@@ -0,0 +1,35 @@
+//! Constants for Block Subsidy, Funding Streams, and Founders’ Reward
+
+use std::time::Duration;
+
+use zebra_chain::{amount::COIN, block::Height};
+
+/// SlowStartInterval
+pub const SLOW_START_INTERVAL: Height = Height(20_000);
+
+// MaxBlockSubsidy
+pub const MAX_BLOCK_SUBSIDY: i64 = (12.5 * (COIN as f64)) as i64;
+
+/// PreBlossomPoWTargetSpacing
+pub const PRE_BLOSSOM_POW_TARGET_SPACING: Duration = Duration::from_secs(150);
+
+/// PostBlossomPoWTargetSpacing
+pub const POST_BLOSSOM_POW_TARGET_SPACING: Duration = Duration::from_secs(75);
+
+/// BLOSSOM_POW_TARGET_SPACING_RATIO = PreBlossomPoWTargetSpacing / PostBlossomPoWTargetSpacing
+pub const BLOSSOM_POW_TARGET_SPACING_RATIO: i64 =
+    (PRE_BLOSSOM_POW_TARGET_SPACING.as_secs() / POST_BLOSSOM_POW_TARGET_SPACING.as_secs()) as i64;
```

Comment:
This could be easier to use as a `u64` - it can't be negative.

Reply:
https://github.com/ZcashFoundation/zebra/pull/1051/commits/364f70eea3f19be9c35d884d861b5096018aa4fd